fe/fe-core/src/main/java/com/starrocks/mysql/privilege/Auth.java

```diff
@@ -1390,6 +1397,39 @@ public TFetchResourceResult toResourceThrift() {
         }
     }
 
+    public  boolean isSupportKerberosAuth() {
+        if (!Config.enable_authentication_kerberos ||
+                Config.authentication_kerberos_service_key_tab.isEmpty() ||
+                Config.authentication_kerberos_service_principal.isEmpty()) {
```

Comment:
Add more concise error messages to dedicate the error.

Reply:
done